controller/resource/metricsresource/metric.go

```diff
@@ -24,8 +24,9 @@ var (
 			Subsystem: PrometheusSubsystem,
 			Name:      "operation",
 			Help:      "Time taken to process a single reconciliation operation.",
+			Buckets:   []float64{1, 2, 5, 10, 20, 40, 60, 120},
```

Comment:
The default buckets are a bit off for the resource implementations. I am not sure what the more proper buckets should look like though, so some feedback would be appreciated. Everything under a second is not really relevant. Everything taking more than 10 seconds is too much. We probably have resources that take up to a minute or two. WDYT?